onnxruntime/core/providers/cpu/object_detection/non_max_suppression_helper.h

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <stdint.h>
+
+#ifdef __NVCC__
+#include "core/providers/cuda/cu_inc/common.cuh"
+#define ORT_DEVICE __device__
+#define HelperMin(a, b) _Min(a, b)
+#define HelperMax(a, b) _Max(a, b)
```

Comment:
What's the reason why we can't use std::min/std::max here?

Reply:
I am new to CUDA programming so I followed the established pattern.

---
In reply to: [298059927](https://github.com/microsoft/onnxruntime/pull/1303#discussion_r298059927) [](ancestors = 298059927)